styles.css

```diff
@@ -1 +1,46 @@
 /* This will be the styles file for Healthy-Restaurant */
+body {
+  margin: 0 160px;
+  background: black;
+  color: white;
+  font-family: "Poppins", sans-serif;
+}
+header {
+  display: flex;
+  height: 80px;
+  justify-content: space-between;
+  align-items: center;
+  padding: 20px 161px 20px 147px;
+}
+.logo-section {
+  display: flex;
+  align-items: center;
+}
+.logo-section p {
+  font-size: 16px;
+  font-weight: 400;
+  margin-left: 9px;
+}
+
+.navbar {
+  display: flex;
+  justify-content: space-evenly;
+  align-items: center;
+  gap: 48px;
+}
+.navbar li {
+  list-style-type: none;
+}
+.navbar li a {
+  text-decoration: none;
+  color: white;
+}
+.navbar button {
+  width: 112px;
+  height: 41px;
```

Comment:
height/width usage should be avoided, lets us padding and font size to match designs